dali/python/nvidia/dali/_debug_mode.py

```diff
@@ -204,8 +202,9 @@ def _get_batch(self, epoch_idx):
 
     def _feed_input(self, data, kwargs):
         if self._callback is not None:
-            raise RuntimeError(f"Cannot use `feed_input` on the external source '{self._name}' with a `source`"
-                               " argument specified.")
+            raise RuntimeError(
+                f"Cannot use `feed_input` on the external source '{self._name}' with a `source`"
+                " argument specified.")
```

Comment:
That's more for future extensions - while you can concatenate f- and non-f strings this way, it might be surprising that formatting wouldn't work in the continuation - and this is going to go unnoticed, just putting the format clause in the output.
```suggestion
                f"Cannot use `feed_input` on the external source '{self._name}' with a `source`"
                f" argument specified.")
```

Reply:
Done